packages/pds/src/db/tables/record.ts

```diff
@@ -7,7 +7,7 @@ export interface Record {
   rkey: string
   repoRev: string | null
   indexedAt: string
-  takedownId: number | null
+  takedownId: string | null
```

Comment:
I have a feeling you're already on it, but just a note to ensure we don't forget the db migration path for these.

Reply:
yup yup 👍